snuba/cli/bootstrap.py

```diff
@@ -5,7 +5,7 @@
 
 
 @click.command()
-@click.option('--bootstrap-server', default=settings.DEFAULT_BROKERS, multiple=True,
+@click.option('--bootstrap-server', default=[], multiple=True,
```

Comment:
Is this code even being used right now?

Reply:
yes, it is used to build the kafka topics and the tables locally.
I believe it is also used in:
> sentry devservices up